<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateDialogUI</class>
 <widget class="QDialog" name="CreateDialogUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crear Tarea</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="nameLayout">
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Nombre:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="commentLayout">
     <item>
      <widget class="QLabel" name="commentLabel">
       <property name="text">
        <string>Comentario:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="commentText"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="comboLayout">
     <item>
      <widget class="QLabel" name="modelLabel">
       <property name="text">
        <string>Modelos:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="modelCombo">
       <item>
        <property name="text">
         <string>Modelo 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Modelo 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Modelo 3</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>188</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
